### DISCUSSION TITLE
How about a docker-compose / yaml validator?

### DISCUSSION THREAD

**Discussion by ovizii** [CorentinTh/it-tools#893] [Ideas]
Something like this: https://codebeautify.org/yaml-validator/

**Comment by sharevb** [comment]
Hi @ovizii, will work on this soon (yaml syntax validator with error details and docker-compose schema validation)

**Reply by sharevb** [reply]
Done in #957

**Comment by sharevb** [comment]
Hi, if you arr interested in up to date version of it-tools, I made kind of a fork here : https://github.com/sharevb/it-tools (https://sharevb-it-tools.vercel.app/ and docker images https://github.com/sharevb/it-tools/pkgs/container/it-tools)
